apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -381,6 +385,10 @@ const Payment = new GQLListSchema('Payment', {
                 }
                 const frozenFields = PAYMENT_FROZEN_FIELDS[oldStatus]
                 for (const field of frozenFields) {
+                    // We need to connect receipt after it's creation, if payment was made for virtual receipt
+                    if (field === 'receipt' && !existingItem.receipt) {
```

Comment:
You should not hard-code it that way

Reply:
Changed to work like PAYMENT_FROZEN_FIELDS